addons/web/__manifest__.py

```diff
@@ -333,6 +333,7 @@
             'web/static/lib/select2-bootstrap-css/select2-bootstrap.css',
             'web/static/lib/daterangepicker/daterangepicker.css',
             'web/static/fonts/fonts.scss',
+            'web/static/src/common/**/*.scss',
```

Comment:
I just left boolean_toggle in `web/static/src/webclient/boolean_toggle/boolean_toggle.scss` since it originates from `webclient.scss` and I don't want it to disappear with legacy :p